transport/transport-api/src/main/java/com/google/android/datatransport/Priority.java

```diff
@@ -14,10 +14,41 @@
 
 package com.google.android.datatransport;
 
+import java.util.Arrays;
+
 public enum Priority {
   /** Quality of the service is within an hour. */
-  DEFAULT,
+  DEFAULT(0),
+
+  /** Events delivered at most daily, on unmetered network. */
+  VERY_LOW(1),
+
+  /** Events delivered as soon as possible. */
+  HIGHEST(2),
+  ;
```

Comment:
remove the `;`

Reply:
It's needed syntactically actually, note the comma one line above. I added it to make these lines friendlier to future diffs

Reply:
ack